hadoop-cloud-storage-project/hadoop-huaweicloud/src/site/markdown/index.md

```diff
@@ -24,12 +24,14 @@ This support comes via the JAR file `hadoop-huaweicloud.jar`.
 
 ## Features
 
-* Read and write data stored in a HuaweiCloud OBS account.
-* Reference file system paths using URLs using the `obs` scheme.
-* Present a hierarchical file system view by implementing the standard Hadoop `FileSystem` interface.
-* Support multipart upload for a large file.
+* Present a hierarchical HDFS file system by implementing the standard Hadoop FileSystem interface.
+* In hadoop, Read and write data stored in a HuaweiCloud OBS bucket.
```

Comment:
Let it be similar to the azure like following..? 

`Read and write data stored in an Azure Data Lake Storage account.`